exonum-java-binding/CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The default [`Transaction#info`][tx-info-07] implementation causing an error on `transaction`
 request. It is modified to return an empty object by default (no info in `content.debug` field
 of the response to `transaction`). (#904)
+- Application packaging issue that might have resulted in several versions of Java artifacts
+on the application classpath.
```

Comment:
PR link is missing

Reply:
It is unknown when one writes the changelog.